app/src/layouts/DashboardLayout/MenuHeader/index.js

```diff
@@ -75,21 +76,24 @@ const MenuHeader = () => {
             >
               <div className="header-left-section">
                 <WorkspaceSelector />
-
-                <a
-                  target="_blank"
-                  rel="noreferrer"
-                  href={LINKS.YOUTUBE_TUTORIALS}
-                  onClick={() => trackTopbarClicked("tutorials")}
-                >
-                  Tutorials
-                </a>
-                <ProductsDropDown />
+                {isAppTypeRequestly && (
+                  <>
+                    <a
+                      target="_blank"
+                      rel="noreferrer"
+                      href={LINKS.YOUTUBE_TUTORIALS}
+                      onClick={() => trackTopbarClicked("tutorials")}
+                    >
+                      Tutorials
+                    </a>
+                    <ProductsDropDown />
+                  </>
+                )}
               </div>
             </Col>
           ) : null}
 
-          {appMode !== GLOBAL_CONSTANTS.APP_MODES.EXTENSION && (
+          {appMode !== GLOBAL_CONSTANTS.APP_MODES.EXTENSION && isAppTypeRequestly && (
```

Comment:
this check would become cleaner if constant is moved to  [`common/constants.js` under `CONSTANTS.APP_MODES`](https://github.com/requestly/requestly/blob/6db0cbf3eb80ea230a530912e54d514122e44b5d/common/constants.js#L10-L16)